psignifit/_posterior.py

```diff
@@ -5,7 +5,7 @@
 import scipy.special as sp
 from scipy import optimize
 
-from ._utils import fp_error_handler, PsignifitException
+from ._utils import fp_error_handler, PsignifitException, cast_np_scalar
```

Comment:
alphabetical order please

Reply:
fixed in 326c7caa01205a68d86463c5bc55402a90b1999a